Rocket.Chat/Controllers/Chat/MessagesListViewController.swift

```diff
@@ -10,7 +10,7 @@ import UIKit
 import RealmSwift
 
 extension APIResult where T == SubscriptionMessagesRequest {
-    func getMessages() -> [Message?]? {
+    func fetchMessagesFromRealm() -> [Message]? {
```

Comment:
What you mean with `fetchMessagesFromRealm()` here? This method isn't getting the values from JSON?

Reply:
@rafaelks It looks a bit confusing, but there's a difference between `fetchMessagesFromRealm` and the `messages` property of the result. The difference is: `messages` just map the values from JSON without using Realm to fetch subobjects that only come with the `primary key` from the server. 

This is why we have this method in the first place, it was there before I started working on it, I just decided to change its name to make it clear that it is not just mapping the JSON, but actually fetching the objects referenced in JSON from Realm. 

In practice, the difference consists in passing the current Realm as a parameter in `Message.map` instead of `nil`.

Reply:
👍